app/base.scss

```diff
@@ -0,0 +1,36 @@
+$brand-primary: #002F2F;
```

Comment:
You're missing few bits that the lab said to include. You need 2 mixin's  and set default styles for your ``html, body, p, pre, h1-h6, button, label, form, ul, ol, li, a``.